code/modules/mob/living/carbon/human/species/zombie.dm

```diff
@@ -59,10 +59,10 @@
 	zombie.equip_to_slot_or_del(new /obj/item/weapon/zombie_claws(zombie), WEAR_L_HAND, TRUE)
 	zombie.equip_to_slot_or_del(new /obj/item/clothing/glasses/zombie_eyes(zombie), WEAR_EYES, TRUE)
 
-	var/datum/disease/black_goo/D = locate() in zombie.viruses
-	if(!D)
-		D = zombie.AddDisease(new /datum/disease/black_goo())
-	D.stage = 5
+	var/datum/disease/black_goo/zombie_infection = locate() in zombie.viruses
+	if(!zombie_infection)
+		zombie_infection = zombie.AddDisease(new /datum/disease/black_goo())
+	zombie_infection.stage = 5 // shouldn't this be 3?
```

Comment:
yes it should be 3 stage i am the one that modified how black goo work and it had 5 stage before.